static/js/spa/hooks/useBreadcrumbs.js

```diff
@@ -0,0 +1,22 @@
+import { useLocation, useRouter } from 'wouter-preact';
+
+export const useBreadcrumbs = () => {
+    const [location, setLocation] = useLocation();
+    const { base } = useRouter();
```

Comment:
no need to check `base` as `useLocation()` will have already removed the base part of the URL